src/libcore/lib.rs

```diff
@@ -198,6 +199,13 @@ pub mod iter;
 pub mod option;
 pub mod raw;
 pub mod result;
+#[unstable(feature = "c_variadic",
```

Comment:
You could leave an empty line before attributes, to make their attachment to `pub mod ffi` a bit clearer.

Reply:
Updated